doc/ref_arch/README.md

```diff
@@ -47,6 +47,22 @@ RA1 is focussing on an OpenStack Reference Architecture that will support VM-bas
 
 The scope of RA2 will be to enable support for containerised and VM-based workloads (i.e. VMs managed by Kubernetes). A thorough gap analysis will need performing to understand the level to which Kubernetes might support traditional VNFs (i.e. to what level can Kubernetes conform to the VIM specification from ETSI NFV v3, or the Vi-Vnfm interface, for example).
 
-RA2 is delivering a Kubernetes Reference Architecture that will be standalone, meaning:
-- If there are any components from RA1 (or any other subsequent RA) that are needed in RA2, they will be explicitly included rather than requiring a Reference Implementation to refer to multiple Reference Architectures
-- The Kubernetes Reference Architecture will also include the Kubernetes cloud providers on which a Reference Implementation can choose to deploy an RA2-conformant Kubernetes
+RA2 is delivering a Kubernetes Reference Architecture that will be standalone, meaning that a Reference Implementation for RA2 can be established solely on the content and requirements present in the RA2 documentation.
+
+Kubernetes can be executed on top of differnt types of underlying infrastrucutre. Examples of those underlying infrastructures are:
```

Comment:
nit: different

Reply:
Fixed in 5fbb5aae1ae52f9d5edc4578b753f03322804632